kernelci/config/__init__.py

```diff
@@ -49,6 +49,8 @@ def get_config_paths(config_paths):
             if os.path.isdir(config_path):
                 config_paths.append(config_path)
                 break
+    elif isinstance(config_paths, str):
```

Comment:
If `configs_paths` was an empty string it'd still get handled by the first  `if` block (line 46) is it an expected behaviour?

Reply:
I guess it would break.  It would also break if it was a floating point number though.